tests/integration/framework/util/parallel.go

```diff
@@ -71,9 +71,10 @@ func NewParallel(t *testing.T, fns ...func(*assert.CollectT)) *ParallelTest {
 
 		wg.Wait()
 
-		for _, collect := range collects {
-			collect.Copy(t)
-		}
+		// for _, collect := range collects {
+		//   TODO: stop using Copy() as it is deprecated and just panics now.
+		//   collect.Copy(t)
```

Comment:
@JoshVanL This method is now deprecated and it just panics. Is this needed still?

Reply:
If errors are correctly being reported when a parallel test fails then it can be safely removed without replacement. Please delete the code instead of commenting it out.

Reply:
I don't know if they are because there is nothing connecting t with collections anymore.

Reply:
This is unrelated to my change I just saw this warning.